app/src/main/java/org/oppia/android/app/help/LoadFAQListFragmentListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.android.app.help
+
+/** Listener for when a selection should result to [FAQListFragment] in tablet devices. */
+interface LoadFAQListFragmentListener {
+  /** Starts [FAQListFragment] in tablet devices. */
```

Comment:
This should describe what the user is doing, not what the implementation will do.

Reply:
Done.